contributing/documentation/standards.rst

```diff
@@ -137,8 +144,13 @@ Files and Directories
 English Language Standards
 --------------------------
 
-* **English Dialect**: use the United States English dialect, commonly called
-  `American English`_.
+Symfony documentation uses the United States English dialect, commonly called
+`American English`_. Disputes over vocabulary will always be resolved using the
+`American English Oxford Dictionary`_ reference. Disputes over neologisms not
+included yet in the dictionary will be resolved by Symfony Documentation maintainers.
```

Comment:
I'm not sure if we need to be so strict here. Personally, I think that the sentence was fine before and we should just add a reference to the dictionary.